src/python/psychrolib.py

```diff
@@ -1459,3 +1466,24 @@ def CalcPsychrometricsFromRelHum(TDryBulb: float, RelHum: float, Pressure: float
     MoistAirVolume = GetMoistAirVolume(TDryBulb, HumRatio, Pressure)
     DegreeOfSaturation = GetDegreeOfSaturation(TDryBulb, HumRatio, Pressure)
     return HumRatio, TWetBulb, TDewPoint, VapPres, MoistAirEnthalpy, MoistAirVolume, DegreeOfSaturation
+
+
+# This section provides support for vectorization through the use of the Numba library (http://numba.pydata.org/)
+# If Numba is not installed, the library just works with scalars and vectorization is simply not supported 
+try:
+    from inspect import isfunction
+    from numba import vectorize, njit
+
+    has_numba = True
+    # Needs to compile as used in vectorized functions below.
+    isIP = njit(isIP)
+    # Vectorise all 'core' functions. 
+    # Utility function are excluded as they are just wrappers.
+    func_list = []
+    for func in list(globals().items()):
+        if isfunction(func[1]) and func[0].startswith(('Get', 'dLnPws_')) and func != 'GetUnitSystem':
+            globals()[func[0]] = vectorize(func[1])
+            func_list.append(func)
+except ImportError:
+    has_numba = False
+    vectorize = None
```

Comment:
What's the purpose of this line? I have the impression that it doesn't do anything.

Reply:
that's right -- dead code I forgot to remove from the previous version. 👍